applications/xr_holoviz/Dockerfile

```
@@ -0,0 +1,53 @@
+
+ARG BASE_IMAGE
+FROM ${BASE_IMAGE} AS run
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+# Install Vulkan SDK and ICD.
+RUN wget -qO- https://packages.lunarg.com/lunarg-signing-key-pub.asc | gpg --dearmor -o /usr/share/keyrings/lunarg-archive-keyring.gpg
+RUN echo "deb [signed-by=/usr/share/keyrings/lunarg-archive-keyring.gpg] https://packages.lunarg.com/vulkan/1.3.204  jammy main" \
+    | tee /etc/apt/sources.list.d/lunarg-vulkan-1.3.204 -jammy.list
```

Typo? `tee: invalid option -- 'j'`

```suggestion
    | tee /etc/apt/sources.list.d/lunarg-vulkan-1.3.204-jammy.list
```

With this update I see an error at the next `apt update`:
```
Reading package lists... Done                               
E: The repository 'https://packages.lunarg.com/vulkan/1.3.204 jammy Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
```

Should we rely on the prior http://packages.lunarg.com/vulkan/lunarg-vulkan-jammy.list ? Is there a reference for 1.3.204? It would be good to use the Vulkan version aligned with Holoscan SDK, but I'm not sure what is available from upstream.